definitions/divorce/json/CaseField/CaseField-deemed-and-dispensed-nonprod.json

```diff
@@ -136,15 +136,15 @@
     "LiveFrom": "12/08/2020",
     "CaseTypeID": "DIVORCE",
     "ID": "LabelPBANumber",
-    "Label": "Fee account number: ${SolicitorFeeAccountNumber}",
+    "Label": "Fee account number: ${FeeAccountReferenceNumber}",
```

Comment:
Just to confirm, General referral also will be using the new approach? :)
This field is used in both "General Referral" and "Service application" event

Reply:
Yes, updated for both general referral payment and confirm service payment